src/proposals/20251002/SparkEthereum_20251002.sol

```diff
@@ -0,0 +1,292 @@
+            max:              10_000,
+            gap:              500,
+            increaseCooldown: 12 hours
+        });
+
+        // Reduce Stablecoin Market Reserve Factors
+        LISTING_ENGINE.POOL_CONFIGURATOR().setReserveFactor(Ethereum.USDC, 1_00);
+        LISTING_ENGINE.POOL_CONFIGURATOR().setReserveFactor(Ethereum.USDT, 1_00);
+
+        // --- Launch Savings v2 Vaults for USDC, USDT, and ETH ---
+        _configureVaultsV2({
+            vault_    : Ethereum.SPARK_VAULT_V2_SPUSDC,
+            supplyCap : 50_000_000e6,
+            minVsr    : 1e27,
+            maxVsr    : TEN_PCT_APY
+        });
+
+        _configureVaultsV2({
+            vault_    : Ethereum.SPARK_VAULT_V2_SPUSDT,
+            supplyCap : 50_000_000e6,
+            minVsr    : 1e27,
+            maxVsr    : TEN_PCT_APY
+        });
+
+        _configureVaultsV2({
+            vault_    : Ethereum.SPARK_VAULT_V2_SPETH,
+            supplyCap : 10_000e18,
+            minVsr    : 1e27,
+            maxVsr    : FIVE_PCT_APY
+        });
+
+        // Onboard SparkLend ETH
+        _configureAaveToken(Ethereum.WETH_SPTOKEN, 50_000e18, 10_000e18 / uint256(1 days));
+
+        // Claim Aave Core aUSDS Rewards
+        MainnetController(Ethereum.ALM_PROXY).grantRole(
+            IALMProxy(Ethereum.ALM_PROXY).CONTROLLER(),
+            Ethereum.SPARK_PROXY
+        );
+
+        address[] memory assets = new address[](1);
+        assets[0] = Ethereum.ATOKEN_CORE_USDS;
+
+        IALMProxy(Ethereum.ALM_PROXY).doCall(
+            AAVE_INCENTIVE_CONTROLLER,
+            abi.encodeCall(
+                IAaveIncentiveController(AAVE_INCENTIVE_CONTROLLER).claimAllRewardsToSelf,
+                (assets)
+            )
+        );
+
+        MainnetController(Ethereum.ALM_PROXY).revokeRole(
+            IALMProxy(Ethereum.ALM_PROXY).CONTROLLER(),
+            Ethereum.SPARK_PROXY
+        );
+
+        // Add transferAsset Rate Limit for SYRUP
+        SLLHelpers.setRateLimitData(
+            RateLimitHelpers.makeAssetDestinationKey(
+                MainnetController(Ethereum.ALM_CONTROLLER).LIMIT_ASSET_TRANSFER(),
+                SYRUP,
+                Ethereum.ALM_OPS_MULTISIG
+            ),
+            Ethereum.ALM_RATE_LIMITS,
+            200_000e18,
+            200_000e18 / uint256(1 days),
+            18
+        );
+
+        // Transfer Share of Ethena Direct Allocation Net Profit to Grove
+        IERC20(Ethereum.USDS).transfer(GROVE_SUBDAO_PROXY, AMOUNT_TO_GROVE);
+
+        // Spark Foundation
+        IERC20(Ethereum.USDS).transfer(Ethereum.SPARK_FOUNDATION, AMOUNT_TO_SPARK_FOUNDATION);
+
+        // Configure Symbiotic Instance
+        _configureSymbiotic();
+
+        // Withdraw USDS and DAI Reserves from SparkLend
+        address[] memory aTokens = new address[](2);
+        aTokens[0] = Ethereum.DAI_SPTOKEN;
+        aTokens[1] = Ethereum.USDS_SPTOKEN;
+
+        _transferFromSparkLendTreasury(aTokens);
```

Comment:
Can we please move these between E3 ("SparkLend - Reduce Stablecoin Market Reserve Factors") and E5 ("Spark Savings v2 - Launch Savings Vaults for USDC, USDT, and ETH") as it is in the post?